src/Lykke.MatchingEngine.Connector/Domain/Fee.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Lykke.MatchingEngine.Connector.Abstractions.Models;
+
+namespace Lykke.MatchingEngine.Connector.Domain
+{
+    public class Fee
```

Comment:
Better to distinguish domain and contract names. FeeContract and Fee, for example

Reply:
Changed to FeeContract